invenio_app_ils/mail/tasks.py

```diff
@@ -30,7 +40,13 @@ def send_ils_email(message, name="ils_mail"):
     log_msg = dict(
         name=name, action="before_send", message_id=dump["id"], data=dump
     )
-    celery_logger.debug(json.dumps(log_msg, sort_keys=True))
+    celery_logger.info(json.dumps(log_msg, sort_keys=True))
+
+    if called_by_celery_task():
```

Comment:
I think that we need to take a decision here (ping @jrcastro2 @kprzerwa too). When the email is sent after a checkout, do we consider it automatic or manual?

IMHO, manual is only when the librarian will click on the button "send email" but in all other cases it should be automatic.
If we go for this solution then I would change this code to be always automatic and then override the param to manual only when coming from the endpoint that is connected to the send mail buttons.
The manual email is sent via the `LoanMailResource` endpoint.

Reply:
I agree with you, I would consider it manual only if the librarian explictly send it.